example/hatchify-app/frontend/App.tsx

```diff
@@ -3,15 +3,17 @@ import { useState } from "react"
 import { v2ToV1 } from "@hatchifyjs/core"
 import {
   hatchifyReact,
-  MuiProvider,
+  HatchifyProvider,
   createJsonapiClient,
 } from "@hatchifyjs/react"
+import { createTheme, ThemeProvider } from "@mui/material"
 import { Todo } from "../schemas/todo"
 import { User } from "../schemas/user"
 
 export const hatchedReact = hatchifyReact(
   createJsonapiClient("http://localhost:3000/api", v2ToV1({ Todo, User })),
 )
+const defaultTheme = createTheme()
```

Comment:
(minor) but to reduce lines in our guide (and this should do it's best to mimic the guide), lets move this to just be inline; `ThemeProvider theme={createTheme()}`